submissions/Jemikalajah-Philip/Html/cart.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
This PR looks generally bloated. Please review and remove any unneeded files. For example, pictures named `toy--` are not used anywhere

Reply:
Please,  these are pictures  that will be used for my design prototyping. It will be implemented  sooner or later.  Hence they're  needed files.

Reply:
keyword, "will be". I don't know this design prototyping task, but it will most likely be done in another folder. So please remove unneeded files

Reply:
Same applies to all `baby--.jpg` files